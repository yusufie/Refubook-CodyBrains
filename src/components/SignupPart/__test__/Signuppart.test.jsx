import React from 'react';
import renderer from 'react-test-renderer';
import SignUp from '../Signuppart';

jest.mock('react-router-dom', () => ({
  Link: 'Link',
}));
jest.mock('react-redux', () => ({
  useSelector: () => ({
    user: {
      name: 'test',
    },
  }),
}));

describe('SignUp', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SignUp />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
