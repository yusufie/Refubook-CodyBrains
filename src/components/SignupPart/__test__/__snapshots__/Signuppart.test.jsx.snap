// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders correctly 1`] = `
<section
  className="flex flex-col"
>
  <div
    className=" mx-20 pb-20 grid place-items-center"
  >
    <img
      alt=""
      className="w-60"
      src="hug.png"
    />
    <h2
      className="text-2xl xl:text-4xl text-refubookBlue mb-4 text-center"
    >
      Share_Your_Story
    </h2>
    <p
      className="text-center pt-7 pb-5 px-15 mx-15 text-refubookGray font-medium  text-l max-w-screen"
    >
      Share_Your_Story_Content
    </p>
    <Link
      to="/profile"
    >
      <button
        className="py-1 px-3 md:py-2 md:px-9 bg-refubookBlue text-refubookWhite font-bold text-lg md:text-xl  rounded-full shadow-lg hover:bg-refubookWhite hover:text-refubookBlack transition duration-500 ease-in-out"
        type="button"
      >
        Create_Blog_Post
      </button>
    </Link>
  </div>
</section>
`;
